pages/common/ansible-inventory.md

```diff
@@ -0,0 +1,21 @@
+# ansible-inventory
+
+> Display or dump an Ansible inventory.
+> See also: `ansible`.
+> More information: <https://www.ansible.com/>.
```

Comment:
I like to remove the trailing slashes for root urls like this one.
```suggestion
> More information: <https://www.ansible.com>.
```

Reply:
This is the same More Information link that is used in the ansible tldr-page. Should I change that one too?

Reply:
I'd use https://docs.ansible.com/ansible/latest/cli/ansible-inventory.html, actually for this page.

>Should I change that one too?

It doesn't matter.  I think it's too minor for you to go out of your way to fix it.